README.md

```diff
@@ -64,6 +64,18 @@ with the broader community is encouraged to join.
 * [Past meeting recordings]
 * [Community Calendar][community calendar]
 
+### Special Interest Groups (SIG)
+Each SIG collaborates in Slack and some groups have regular meetings, you can find the meetings in the [Community Calendar][community calendar].
```

Comment:
Could you wrap at 80 chars to match the rest of of the doc, please.

Reply:
Ping. Waiting for this before merging. 😄

Reply:
sure @negz 👍

Reply:
sorry that I didn't see the notification for this @negz 
I made the changes